src/main/java/org/jboss/pull/player/GitHubApi.java

```diff
@@ -211,15 +170,15 @@ private List<ModelNode> filterNonModifiedPullRequests(List<ModelNode> pulls, Ins
         return res;
     }
 
-    private String nextLink(HttpResponse response) {
-        Header header = response.getFirstHeader("Link");
-        if (header == null) { // no pagination
-            return null;
-        }
-        for (HeaderElement el : header.getElements()) {
-            if ("next".equals(el.getParameterByName("rel").getValue())) {
-                String link = el.getName() + "=" + el.getValue();
-                return link.substring(1, link.length() - 1);
+    private String nextLink(HttpResponse<?> response) {
+        final Optional<String> linkHeader = response.headers().firstValue("Link");
+        if (linkHeader.isPresent()) {
+            for (String headerValue : response.headers().allValues("Link")) {
+                System.out.printf("Link Value: %s%n", headerValue);
+                /*if ("next".equals(el.getParameterByName("rel").getValue())) {
```

Comment:
Is this return not needed?

Reply:
Oops, I missed that as I need to parse it. I was just trying to see what the values were. Nice catch!

Reply:
Should be fixed now.